staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -107,6 +108,7 @@ func newGenericWebhook(scheme *runtime.Scheme, codecFactory serializer.CodecFact
 	clientConfig.ContentConfig.NegotiatedSerializer = serializer.NegotiatedSerializerWrapper(runtime.SerializerInfo{Serializer: codec})
 
 	clientConfig.Dial = customDial
+	clientConfig.Wrap(x509metrics.NewMissingSANRoundTripperWrapperConstructor(x509MissingSANCounter))
```

Comment:
Have we considered setting the `Warning` header?

Reply:
I don't think I would like to alert every user of the cluster about its internals.